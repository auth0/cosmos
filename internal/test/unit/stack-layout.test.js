import React from 'react'
import { shallow } from 'enzyme'
import { StackLayout } from '@auth0/cosmos'

describe('Stack Layout', () => {
  it('renders correctly when a child is null and it has a space assigned', () => {
    const stackLayout = shallow(
      <StackLayout space={['none', 1]}>
        {null}
        <div>something</div>
      </StackLayout>
    )
    expect(stackLayout).toMatchSnapshot()
  })
})
