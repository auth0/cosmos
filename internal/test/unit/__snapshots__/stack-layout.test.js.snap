// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stack Layout renders correctly when a child is null and it has a space assigned 1`] = `
<styled.div
  alignment="center"
  data-cosmos-key="stack-layout"
  distribution="start"
  gutter="medium"
  space={
    Array [
      "none",
      1,
    ]
  }
>
  <styled.div
    className="cosmos-layout-child"
    data-cosmos-key="stack-layout.item"
    key=".$1"
    space={1}
  >
    <div>
      something
    </div>
  </styled.div>
</styled.div>
`;
